calcom/cal.com
# feat: option for adding ics events to workflow reminders

## What does this PR do?

This PR adds a feature that lets users include a calendar event (ics file) to workflow reminder emails.

<img width="784" alt="image" src="https://github.com/calcom/cal.com/assets/138862352/1b106214-040a-41d5-9430-e418fb323e83">


## Type of change


## How should this be tested?

1. Go to "Workflows"
2. Create a workflow reminder having an email trigger with the "Include Calendar Event" option checked.
3. Workflow reminder Emails should now include calendar events when delivered.

## Mandatory Tasks


## Checklist



- I have read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code does follow the style guidelines of this project
- I have commented my code, particularly in hard-to-understand areas
- I have checked if my PR needs changes to the documentation
- I have checked if my changes generate no new warnings
